### ISSUE TITLE
external storage documentation very lacking

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#410]
Originally reported of Google Code with ID 354.

```
extstorage has a manpage for the interface, but is not mentioned among the storage backends in gnt-instance nor explained well in a design doc or other man page to allow easy understanding and usage.
```
Originally added on 2013-01-31 12:39:49 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi Constantinos,

As we chatted on the list already about this not being documented, I am assigning the relevant Issue to you. I hope we can ship extstorage with a few docs, for 2.7!

Thanks a lot,

Guido

```
Originally added on 2013-02-11 15:38:19 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with commit e19f7095c18f3cfd0f9ba451ca82bbbffc16bd74
```
Originally added on 2013-02-19 15:07:38 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
